lib/cmake/scipp-conan.cmake

```diff
@@ -16,10 +16,10 @@ if(NOT EXISTS "${CMAKE_CURRENT_BINARY_DIR}/conan.cmake")
   )
   file(
     DOWNLOAD
-    "https://raw.githubusercontent.com/conan-io/cmake-conan/0.17.0/conan.cmake"
+    "https://raw.githubusercontent.com/conan-io/cmake-conan/6e5369d13720f22e07e31bb6b9018dbe60529fea/conan.cmake"
```

Comment:
What version is this?

Reply:
The current HEAD of the main branch for conan1.X, there hasn't been a release in some time.